portfolio/src/main/webapp/index.html

```diff
@@ -33,7 +33,7 @@ <h1>Welcome to BetterBudget!</h1>
                 <a class="nav-link" href="emergencyFund.html">Emergency Fund</a> <!--To Emergency Fund Calculator-->
               </li>
               <li class="nav-item">
-                <a class="nav-link" href="#"> Rent vs. Buy</a>
+                <a class="nav-link" href="buy_rent.html"> Rent vs. Buy</a> <!--To Rent vs Buy Calculator-->
```

Comment:
Can you update this on [the other navbar](https://github.com/jdtyner/sps22-team6/blob/d6aebf1dbd0362c8f64894c2da6a7dd6223b037a/portfolio/src/main/webapp/income-expense.html#L33) too

Reply:
I can update that